packages/vuetify/src/components/VChip/VChip.tsx

```diff
@@ -142,10 +152,10 @@ export const VChip = defineComponent({
           disabled={ props.disabled || undefined }
           draggable={ props.draggable }
           href={ link.href.value }
-          v-ripple={ [isClickable && props.ripple, null] }
-          onClick={ isClickable && onClickFunc }
+          v-ripple={ [isClickable.value && props.ripple, null] }
+          onClick={ isClickable.value && onClick }
```

Comment:
There's already an isClickable check in the function

```suggestion
          onClick={ onClick }
```